test/object-store/sync/flx_sync.cpp

```diff
@@ -2916,17 +2956,26 @@ TEST_CASE("flx: data ingest", "[sync][flx][data ingest][baas]") {
 
             auto table = realm->read_group().get_table("class_Asymmetric");
             REQUIRE(table->size() == 0);
+
+            auto documents = get_documents(*realm->config().sync_config->user, "Asymmetric", 100);
+            for (int i = 0; i < 100; ++i) {
+                check_document(documents, obj_ids[i], {{"location", util::format("foo_%1", i)}});
+            }
         });
     }
 
     SECTION("create multiple objects - same commit") {
         harness->do_with_new_realm([&](SharedRealm realm) {
             CppContext c(realm);
             realm->begin_transaction();
+            std::vector<ObjectId> obj_ids;
             for (int i = 0; i < 100; ++i) {
-                auto obj_id = ObjectId::gen();
+                obj_ids.push_back(ObjectId::gen());
                 Object::create(c, realm, "Asymmetric",
-                               std::any(AnyDict{{"_id", obj_id}, {"location", util::format("foo_%1", i)}}));
+                               std::any(AnyDict{
+                                   {"_id", obj_ids.back()},
+                                   {"location", util::format("foo_%1", i)},
```

Comment:
can we change the location here (i.e, "bar_%1") so it does not overlap with the objects created in the previous section?